modules/storage/src/main/java/com/gluonhq/attach/storage/impl/AndroidStorageService.java

```diff
@@ -38,15 +38,14 @@
 public class AndroidStorageService implements StorageService {
 
     private static final Logger LOG = Logger.getLogger(AndroidStorageService.class.getName());
-    private static final boolean debug = Boolean.getBoolean(Constants.ATTACH_DEBUG);
 
     static {
         System.loadLibrary("Storage");
     }
 
     public AndroidStorageService() {
-        if (debug) {
-            enableDebug();
+        if (Boolean.getBoolean(Constants.ATTACH_DEBUG)) {
```

Comment:
Why not use `Util.isDebug()`?

Reply:
That's a different `Util`, it is `"Dalvik"Util`